src/routes/InventoryTable.js

```diff
@@ -107,18 +107,26 @@ const Inventory = ({
             options.filters = Object.entries(defaultFilters).map(([key, val]) => ({ [key]: val }));
         }
 
+        let results = [];
+        options?.filters.forEach((item)=>{
```

Comment:
Why not `.filter` the filters (and avoid `let` and filling it)?

Reply:
Honestly I just thought it looked better. With the way the data is formatted I can do something like
        let results = options?.filters.filter((item)=> item.osFilter);
but then in the operatingSystem conditional on line 129, itd have to be 
results2[0]?.osFilter 
and I thought that looked wonky